lib/seahorn/BvOpSem2RawMemMgr.cc

```diff
@@ -351,10 +352,14 @@ PtrTy RawMemManager::ptrAdd(PtrTy ptr, Expr offset) const {
 }
 
 /// \brief Stores an integer into memory
-///
+/// \param[in] ptr is the address at which _val will be stored
+/// \param[in] _val is the value being written
+/// \param[in] mem is the memory bank/register being written to
+/// \param[in] ty is the type of value being stored
+/// \param[in] align is the known alignment of the ptr
 /// Returns an expression describing the state of memory in \c memReadReg
 /// after the store
-/// \sa loadIntFromMem
+/// \sa storeIntFromMem
```

Comment:
this is wrong. This is already `storeIntToMem`. The additional function to **see also** is `loadIntFromMem`

Reply:
fixed typo